Week1/homework/Masoud/js-exercises/EX1-helloWorld.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
Maybe add a new line after `use strict` to have a separation between that and the rest